overlays/custom-packages/labwc/default.nix

```diff
@@ -5,6 +5,7 @@
 #
 (final: prev: {
   labwc = prev.labwc.overrideAttrs (prevAttrs: {
+    patches = [./labwc-colored-borders.patch];
```

Comment:
We can do this labwc patching in `packages/labwc/default.nix` instead.

Reply:
@vilvo I am not sure I understand what is `packages/labwc/default.nix` that you mentioned. In my opinion, this overlay seems to be the best place for keeping the patch.